integration-tests/java/src/integration-test/java/com/stellarstation/api/test/satellite/SatelliteStreamerTest.java

```diff
@@ -124,19 +115,83 @@ public void onCompleted() {}
             .setEnableEvents(true)
```

Comment:
I guess you can remove setEnableEvents here (it doesn't do anything but isn't intended to be set for this test).